packages/manager/src/components/CopyableTextField/CopyableTextField.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { renderWithTheme } from 'src/utilities/testHelpers';
+import { CopyableTextField } from './CopyableTextField';
+import { downloadFile } from 'src/utilities/downloadFile';
+import { CopyableTextFieldProps } from './CopyableTextField';
+import userEvent from '@testing-library/user-event';
+
+vi.mock('src/utilities/downloadFile', () => ({
+  downloadFile: vi.fn(),
+}));
+
+const mockLabel = 'label';
+const mockValue = 'Text to Copy';
+const defaultProps: CopyableTextFieldProps = {
+  label: mockLabel,
+  value: mockValue,
+};
+
+const setup = (props = {}) => {
```

Comment:
We should improve the type safety here. This seems to allow us to pass any object with any values in it

```suggestion
const setup = (props: Partial<CopyableTextFieldProps> = {}) => {
```

**but,** (see my other comment)